ext/server/device_impl.cpp

```diff
@@ -1054,7 +1059,7 @@ Tango::ConstDevString Device_3ImplWrap::dev_status()
     {
         if (override dev_status = this->get_override("dev_status") )
 	{
-            std::string status = dev_status();
+            std::string status = boost::python::call<std::string>(this->get_override("dev_status")());
```

Comment:
Is the assignment to `dev_status` on line 1060 still required?  Not sure how `override` works in this context - is it just a type?
Could we do `boost::python::call<std::string>(dev_status())`

Reply:
Seems that this change is not correct. Travis is failing due to this I believe.
Do you remember why you did this @NexeyaSGara ?

Reply:
Should be ok now

Reply:
Should be ok now